### PR TITLE
Add documentation for initial commands

### PR DESCRIPTION
**Pull request by gcpinckert**
Adds documentation for the following:

- What is required to run edamame?
- How to install and configure edamame?
- The current usage of available commands.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,105 @@
...
+
+- An [AWS Account](https://docs.aws.amazon.com/SetUp/latest/UserGuide/setup-prereqs-instructions.html)
+- The [AWS CLI tool](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-getting-started.html) should be installed and configured.
+- [`kubectl`](https://kubernetes.io/docs/tasks/tools/#kubectl), a command line tool for working with Kubernetes clusters, should be installed.
+- [`eksctl`](https://docs.aws.amazon.com/eks/latest/userguide/eksctl.html), a command line tool for working with EKS clusters, should be installed.
+
+## Installation
+
+- Clone the repository
+- In the root directory, run the command `npm install` followed by the command `npm init`.
```
Comment (lukeoguro) [inline README.md]: I believe this is `npm link`!

File: README.md [hunk 2]
```diff
@@ -1 +1,105 @@
...
+
+Usage: `edamame get`
+Outputs:
+
+```
+✔ Retrieving all test ids...
+[{"id":1},{"id":2},{"id":3}]
+```
+
+- Retrieves all the ids for tests that have been run in JSON format.
```
Comment (lukeoguro) [inline README.md:87]: Can you make a note that this feature will also be deprecated in favor of test names or randomly generated uuids?

**Review by lukeoguro — APPROVED** [review]
A few minor comments — great documentation!

File: README.md (hunk 2 above)
Comment (gcpinckert) [reply README.md:87]: @lukeoguro  Do you mean the entire feature of getting the ids will be removed, or just that the format of the ids may change (to user generated test names/UUIDs)?

**Pull request merged**